pheweb/serve/server_jeeves.py

```diff
@@ -196,9 +202,150 @@ def get_single_variant_data(self, variant: Variant)-> Tuple[Variant, List[PhenoR
         else:
             return None
 
+    def add_annotations(self, chr, start, end, datalist):
+        if chr == 'X':
+            chr = 23
+        if start == 0:
+            start = 1
+        t = time.time()
+        # TODO tabix fetch takes forever, combine FG and gnomAD annotations and use relevant columns only, or get annotations on the fly for individual variants
+        annotations = self.annotation_dao.get_variant_annotations_range(chr, start, end)
+        annot_hash = { anno.varid: anno.get_annotations() for anno in annotations }
+        gnomad = self.gnomad_dao.get_variant_annotations_range(chr, start, end)
+        gnomad_hash = { anno.varid: anno.get_annotations() for anno in gnomad }
+        print("getting annotations for {} bp took {} seconds".format(end-start+1, time.time()-t ) )
+        for d in datalist:
+            print('n variants ' + str(len(d['data']['id'])))
```

Comment:
I think we should be switching to using dataclasses instead of dicts. Not necessary to start changing now but let's start gradually doing that. 
https://dev.to/victorosilva/explicit-is-better-than-implicit-avoid-dict-overuse-57k4

Reply:
Yes